homeassistant/components/light/zwave.py

```diff
@@ -217,11 +217,10 @@ def _get_color_values(self):
                 self._value_color = value_color
 
         if self._value_color_channels is None:
-            for value_color_channels in self._value.node.get_values(
-                    class_id=zwave.const.COMMAND_CLASS_SWITCH_COLOR,
-                    genre=zwave.const.GENRE_SYSTEM,
-                    type=zwave.const.TYPE_INT).values():
-                self._value_color_channels = value_color_channels
+            self._value_color_channels = self.value_handler(
```

Comment:
[Right, this is where it's used.](https://github.com/turbokongen/home-assistant/blob/93fe92117838d0a4abb49f6d9a89c5c7bc20b7ec/homeassistant/components/light/zwave.py#L248)